backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8-alpine3.10
+
+RUN apk add --no-cache --virtual .build-deps gcc libc-dev make \
+    && pip install pipenv
+
+COPY Pipfile Pipfile
+COPY Pipfile.lock Pipfile.lock
+
+RUN pipenv install --dev
+
+COPY src /app
+
```

Comment:
you might want to add `EXPOSE 80` to make it verbose that the app runs on this port

Reply:
I've added explicit EXPOSE in 15902f0 .